configs/katello_37.yaml

```diff
@@ -0,0 +1,190 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.6.0:
+    :redmine_release_id: 284
+  :3.6.1:
+    :redmine_release_id: 352
+:releases:
+  :3.7.0:
+    :redmine_release_id: 338
+:code_name: Blonde Ale
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.7
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-selinux:
+    :branch: 3.0.3
+    :repo: https://github.com/Katello/katello-selinux.git
+  :katello-installer:
+    :branch: KATELLO-3.7
+    :repo: https://github.com/Katello/katello-installer.git
+    :version_branch: true
+  :hammer-cli-katello:
+    :branch: 0.12.1
```

Comment:
@akofink Is this the expected version of hammer-cli-katello?

Reply:
No, it'll be 0.13, but we haven't released yet because the hammer-cli-katello tests are failing. re: https://github.com/Apipie/apipie-rails/pull/614